vars/checkPatch.groovy

```diff
@@ -51,8 +52,14 @@ def call(Map config = [:]) {
         ignored_files += ":" + config['ignored_files']
     }
 
+    def checkpatch_args="--notree --show-types -"
+    if (config['codespell_file']) {
+       checkpatch_args="--notree --show-types --codespell --codespellfile=" + config['codespell_file'] + " -"
```

Comment:
```suggestion
       checkpatch_args += "-codespell --codespellfile=" + config['codespell_file'] + " -"
```
Just to DRY up a little bit.

Reply:
The way these args are set is a bit messed up, but the - means "stdin" so has to come last in the list.

Reply:
Indeed.  My suggestion preserves that if I am not mistaken.

But in any case, you are proposing abandoning this for daos-stack/code_review#21 correct?

Reply:
I think that approach works better overall, yes.